test_files/arrow_fn/arrow_fn.js

```diff
@@ -1,8 +1,8 @@
-goog.module('test_files.arrow_fn.arrow_fn');var module = module || {id: 'test_files/arrow_fn/arrow_fn.js'};/**
+goog.module('test_files.arrow_fn.arrow_fn');var module = module || {id: 'test_files/arrow_fn/arrow_fn.js'};
+/**
  * @fileoverview added by tsickle
  * @suppress {checkTypes} checked by tsc
  */
-
```

Comment:
I think this change is because of the missing `EmitFlags.CustomPrologue` that we are using in transformer_util.ts.